packages/desktop-gui/src/project/project-model.js

```diff
@@ -72,8 +72,11 @@ export default class Project {
   @computed get displayName () {
     if (this.name) return this.name
 
-    let splitName = _.last(this.path.split('/'))
-    return _.truncate(splitName, { length: 60 })
+    // need normalize windows paths with \ before split
+    const normalizedPath = this.path.replace(/\\/g, '/')
+    const lastDir = _.last(normalizedPath.split('/'))
```

Comment:
Still not sure why we are normalizing this. It doesn't appear that you're using `projectName` when we still have this special function to carve it out (?)

Does this function need to exist any more?

Reply:
It would be a fallback in case projectName were null, which I'm not sure could happen.

Reply:
It cannot happen but fine... we'll leave it in. Might have been good to leave a note at the top of the function with that uncertainty